src/runner.py

```diff
@@ -1,35 +1,60 @@
 import os
-
 import torch
 
+from src.static.cm_data_aggregate_kenya import KenyaDataAggregator
 from src.gradient.sensitivity_calculator import SensitivityCalculator
 from src.plotter import Plotter
 from src.static.dataloader import DataLoader
+from src.contact_manipulation import ContactManipulation
 
 
 class Runner:
-    def __init__(self, data: DataLoader, model: str):
+    def __init__(self, data: DataLoader, epidemic_model: str, model: str):
         """
         Initialize the simulation with the provided data.
         Args: data (DataLoader): DataLoader object containing age data and model params.
         """
 
         self.data = data
-        self.population = self.data.age_data
-        self.n_age = len(self.data.age_data)
+        if model == "kenya":
```

Comment:
Why do you have this code line here? These are executed in `SensitivityCalculator`. Remove all line which are already included in the constructor of `SensitivityCalculator` and if you need some variables, then get those from there or just simply name the variable for the calculator instance with a shorter name e.g. `sens_calc` and refer to all variables later as `self.sens_calc.n_age` or `self.sens_calc.epidemic_model`. Having duplicated lines in the code makes the project low quality.